EasyPost/Parameters/V2/Base.cs

```diff
@@ -0,0 +1,93 @@
+using EasyPost._base;
```

Comment:
Why are there ultimately two of these same files? One in Beta and one in V2? Since an API version will always be tied to the lib, we should probably flatten this structure and remove the `V2` directory completely. Whatever the lib version is will always use the current API version.

Reply:
I was unaware that was the official approach we had decided on for the eventual API versioning. Will do